modules/Behavior.js

```diff
@@ -68,6 +80,47 @@
     },
 
     /**
+     * This is an overridable method that is called after the view's initialize has completed.
+     * @method _postinitialize
+     * @private
+     */
+    _postinitialize: _.noop,
+
+    /**
+     * This is called after the view's initialize method is called and will wrap the view's prepare()
+     * such that it returns the combination of the view's prepare result with the behavior's prepare result
+     * inside it under the behavior's alias.
+     * This method will call _postinitialize when its finished.
+     * @method __postinitialize
+     * @private
+     */
+    __postinitialize: function() {
+      var originalViewPrepareFn = _.bind(this.view.prepare, this.view);
+      var wrappedPrepareFn = _.wrap(originalViewPrepareFn, this.__viewPrepareWrapper);
+      this.view.prepare = _.bind(wrappedPrepareFn, this);
```

Comment:
do we want the bound/wrapped function to have the behavior as the context of the view? I would have thought the view